pkg/sanity/controller.go

```diff
@@ -210,6 +222,7 @@ var _ = DescribeSanity("Controller Service", func(sc *SanityContext) {
 				},
 			)
 			Expect(err).NotTo(HaveOccurred())
+			cl.RegisterVolume(name, VolumeInfo{VolumeID: vol.GetVolume().GetId()})
```

Comment:
Should this go after L228 because it requires vol.GetVolume().GetId()?

Reply:
Yes, will change that.